Index: konversation/src/config/preferences.cpp
===================================================================
--- konversation.orig/src/config/preferences.cpp
+++ konversation/src/config/preferences.cpp
@@ -69,7 +69,7 @@ Preferences::Preferences()
     serverGroup->addServer(server);
     serverGroup->setIdentityId(mIdentity->id());
     Konversation::ChannelSettings channel;
-    channel.setName(QStringLiteral("#konversation"));
+    channel.setName(QStringLiteral("#kubuntu"));
     serverGroup->addChannel(channel);
     serverGroup->setExpanded(false);
     mServerGroupHash.insert(0, serverGroup);
