Index: konversation/src/config/preferences.cpp
===================================================================
--- konversation.orig/src/config/preferences.cpp
+++ konversation/src/config/preferences.cpp
@@ -62,14 +62,14 @@ Preferences::Preferences()
     mIdentity->setNicknameList(nickList);
 
     Konversation::ServerGroupSettingsPtr serverGroup(new Konversation::ServerGroupSettings);
-    serverGroup->setName(QStringLiteral("freenode"));
+    serverGroup->setName(QStringLiteral("Ubuntu IRC"));
     Konversation::ServerSettings server;
-    server.setHost(QStringLiteral("chat.freenode.net"));
+    server.setHost(QStringLiteral("irc.ubuntu.com"));
     server.setPort(8001);
     serverGroup->addServer(server);
     serverGroup->setIdentityId(mIdentity->id());
     Konversation::ChannelSettings channel;
-    channel.setName(QStringLiteral("#konversation"));
+    channel.setName(QStringLiteral("#kubuntu"));
     serverGroup->addChannel(channel);
     serverGroup->setExpanded(false);
     mServerGroupHash.insert(0, serverGroup);
